src/eq.js

```diff
@@ -0,0 +1,261 @@
+   *      mouseOnHandle(this.x,this.y) ? (this.y=mouseY, makeAdjustment(this.ind,this.y,this.x)) : true;
+   *    } else{
+   *      mouseOnHandle(this.x,this.y) ? (this.x=mouseX, this.y=mouseY, makeAdjustment(this.ind,this.y,this.x)) : true;
+   *    }
+
+   *    if (this.y < 1) {this.y = 1;}
+   *    else if (this.y >255) { this.y = 255;}
+   *    else if (this.x < 1) { this.x = 1;}
+   *    else if (this.x > 709) { this.x = 709;} 
+   *  }
+   *  this.toggle = function () {
+   *    eq.toggleBand(this.ind);
+   *    eq.bands[this.ind].toggle ? this.c = color(255) : this.c = color(30);
+   *  }
+   * }
+
+
+
+   *  function mouseClicked(){
+   *   //console.log(cpts.length);
+   * for (var i = 0; i < cpts.length; i++) {
+    
+   * if (mouseX > cpts[i].x - circleSize && mouseX < cpts[i].x + circleSize &&
+   *       mouseY > cpts[i].y - circleSize && mouseY < cpts[i].y + circleSize){
+   *    cpts[i].toggle();
+   *  } 
+   * }
+      
+   *  }
+
+   *  function makeAdjustment(band,y,x){
+   *  eq.setBand(band,"mod",map(y,256,0, -40, 40), map(x,0,710,0,22050));
+   * }
+
+   * function mouseOnHandle(x,y){
+   *    if (mouseX > x - circleSize && mouseX < x + circleSize &&
+   *     mouseY > y - circleSize && mouseY < y + circleSize &&
+   *     mouseIsPressed){
+   *      return true;
+   *    }
+   * else{
+   *    return false;
+   *  }
+  * }
+   *  </code></div>
+   */
+
+
+  p5.EQ = function(_eqsize) {
+
+    //default size is 8 band EQ
+    _eqsize = _eqsize || 8;
+
+    Effect.call(this);
+    
+    this.bands = [];
+
+    /**
+      *  The p5.EQ is built with
+      *  <a href="http://www.w3.org/TR/webaudio/#BiquadFilterNode">
+      *  Web Audio BiquadFilter Node</a>.
+      *  
+      *  @property biquadFilter
+      *  @type {Object}  Web Audio Delay Node
+      *
+      *  @{param} toggle {boolean} On/of switch for band, true == on
+    */
+    for (var i = 0; i<_eqsize; i++){
+      this.bands.push(this.ac.createBiquadFilter());
+      this.bands[i].type = 'peaking';
+      this.bands[i].frequency.value = i*3150;
+      this.bands[i].Q.value = 5;
+      this.toggle = true;
+      if (i>0){
+        this.bands[i-1].connect(this.bands[i]);
+      }
+    }
+    this.input.connect(this.bands[0]);
+    this.bands[_eqsize-1].connect(this.output);
+  };
+
+  p5.EQ.prototype = Object.create(Effect.prototype);
+
+  p5.EQ.prototype.process = function (src) {
+    src.connect(this.input);
+  };
+
+  /**
+   *  @{method} setBand Make adjustments to individual bands of the EQ
+   *
+   *  @{param} band {number} Band to be modified
+   *  @{param} option {string} Modify function (toggle, mod, type);
+   *  @{param} param1 {number} Set the frequency of a band w/ usage: "mod"
+   *  @{param} param2 {number} Set the Q value (resonance) of a band w/usage: "mod"
+   *  @{param} param1 {string} Set the type of the band filter w/ usage: "type"
+   */
+  p5.EQ.prototype.setBand = function (band, option, param1, param2) {
```

Comment:
we could remove the gain nodes from p5.Filters-- but ultimately the difference in the API is that users would modify bands of the EQ either by 
- `eq.bands[i].set() or .freq() or .res()` <- the established filter API
or
- `eq.setBand(i,"mod",-40)` <- to change the gain of the ith band to -40
to me, it seems more straightforward to use a new .setBand() method from the highest level, rather than requiring users to access elements of the array of band filters. It might still be good to use p5.Filters internally, just to make things clearer for anyone looking at the library.